### PR TITLE
test(clerk-js): Remove skipped integration tests related to navigation

### PR DESCRIPTION
**Pull request by panteliselef**
## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

File: integration/tests/navigation.test.ts [hunk 1]
```diff
@@ -91,129 +91,130 @@ export default function Page() {
...
+
+    await u.page.waitForSelector('.cl-profileSectionContent__username .cl-headerTitle', { state: 'detached' });
+
+    await u.po.userProfile.clickAddEmailAddress();
+
+    u.page.getByText(/an email containing/i);
+
+    await u.po.userProfile.typeEmailAddress('some@email.com');
+
+    await u.page.getByText(/Cancel/i).click();
```
Comment (panteliselef) [inline integration/tests/navigation.test.ts:120]: This test no longer tests navigation but interaction within the component. Should this be moved ?

File: integration/tests/navigation.test.ts [hunk 2]
```diff
@@ -91,129 +91,130 @@ export default function Page() {
...
```
Comment (panteliselef) [inline integration/tests/navigation.test.ts:149]: This test no longer tests navigation but interaction within the component. Should this be moved ?

File: integration/tests/navigation.test.ts [hunk 3]
```diff
@@ -91,129 +91,130 @@ export default function Page() {
...
+
+    await u.page.waitForSelector('.cl-modalContent > .cl-userProfile-root', { state: 'attached' });
+
+    await u.po.userProfile.clickSetUsername();
+    await u.page.getByText(/Cancel/i).click();
+
+    await u.page.waitForSelector('.cl-profileSectionContent__username .cl-headerTitle', { state: 'detached' });
+
+    await u.po.userProfile.clickAddEmailAddress();
+    await u.page.getByText(/Cancel/i).click();
```
Comment (panteliselef) [inline integration/tests/navigation.test.ts:204]: This test no longer tests navigation but interaction within the component. Should this be moved ?

File: integration/tests/navigation.test.ts (hunk 1 above)
Comment (nikosdouvlis) [reply integration/tests/navigation.test.ts:120]: Let's extract this into a UP-specific test file :)

**Review by nikosdouvlis — APPROVED** [review]

**Pull request merged**